src/diffusers/pipelines/stochatic_karras_ve/pipeline_stochastic_karras_ve.py

```diff
@@ -21,13 +22,20 @@ class KarrasVePipeline(DiffusionPipeline):
     unet: UNet2DModel
     scheduler: KarrasVeScheduler
 
-    def __init__(self, unet, scheduler):
+    def __init__(self, unet: UNet2DModel, scheduler: KarrasVeScheduler):
         super().__init__()
         scheduler = scheduler.set_format("pt")
         self.register_modules(unet=unet, scheduler=scheduler)
 
     @torch.no_grad()
-    def __call__(self, batch_size=1, num_inference_steps=50, generator=None, output_type="pil", **kwargs):
+    def __call__(
+        self,
+        batch_size: Optional[int] = 1,
+        num_inference_steps: Optional[int] = 50,
```

Comment:
These shouldn't be optional, as `None` isn't a valid value for them

```suggestion
        batch_size: int = 1,
        num_inference_steps: int = 50,
```